### ISSUE TITLE
Re-evaluate connector OOTB image versioning

### ISSUE THREAD

**Issue by menski** [camunda/connector-sdk#128]
__What should we do?__

We should discuss how we want to further ship the OOTB connectors docker image, and if our current approach is sensible moving forward. 

One approach would be the create a separate github repository and/or docker hub repository to separate the connectors runtime from the OOTB bundle.

__Why should we do it?__

With PR #126 a docker image was added which bundles all OOTB connectors into a single image. This image is based on the base job worker runtime image introduced in #115. The OOTB image is a bundle of a runtime version plus a version of each connector. At the moment the OOTB image is using the same docker image name `camunda/connectors` and the version of the job worker runtime base image with the `ootb` flavor, i.e. the first release is `camunda/connectors:0.2.0-ootb`.

The problem with this approach is that the OOTB is a composite of many versions, i.e. job worker runtime, slack connector, sengrid connector etc. Therefore it is questionable which version the OOTB image should have. It is unclear how we will support the following scenarios:

1. A new version of an existing connector is release
    1. will a new OOTB image be pushed containing the new connector version
    2. what will be the version of the image? The same version of the runtime, i.e. overwriting the existing docker hub tag. This can lead to the fact that different systems have different version of the same docker image tag.
2. A new connector is release
    1. will a new OOTB image be pushed containing the new connector
    2. what will be the version of the image? The same version of the runtime, i.e. overwriting the existing docker hub tag. This can lead to the fact that different systems have different version of the same docker image tag.

**Comment by nikku** [comment]
I tend to agree that we shall version the OOTB connectors image independently from the SDK and connector run-time.

* Pick any valid semantic version start number (i.e. v0.3.0)
* Follow along that scheme and increment semantically with new additions (connectors, bug fixes, connector-sdk, ...)
* Continuously release with new OOTB connectors / SDK releases

**Comment by menski** [comment]
> I tend to agree that we shall version the OOTB connectors image independently from the SDK and connector run-time.

I assume that you then also agree that we should use a different docker image name instead of `camunda/connectors`? Otherwise we have something like
- `camunda/connectors:0.2.0` -> Job Worker Runtime 0.2.0
- `camunda/connectors:0.3.0-ootb` -> Job Worker Runtime 0.2.0 plus other connectors

**Comment by nikku** [comment]
> I assume that you then also agree that we should use a different docker image name instead of camunda/connectors? 

Yes. Maybe `camunda/connectors-ootb`?

**Comment by PreetiNirwal** [comment]
From reading this, it seems we need a bit of de-coupling so we can be more flexible, which should ultimately provide an easier experience when developers are installing OOTB Connectors. 
The de-coupling will also help Camunda when releasing net new items or updates to existing items.

Based on this, I think we should probably separate the runtime, from the bundle of connectors (code).  

Now would this mean that we continue to have 1 OOTB image (with a version), and it could contain different versions of a connector and its associated run-time? If yes, that should be ok. As long as we document it somewhere this isn't a problem correct?

If this is ok & preferred, then we should change the image name to be more clear. I like Nico's suggestion, it's more explicit. 

What do you think? Are my assumptions accurate?

**Comment by menski** [comment]
Yes your assumptions are accurate, I think we all agree on the way forward to decouple these two artifacts. I will setup a new repository on github and docker hub. Regarding the name of the docker image, I guess your proposal is more accessible to uses then `ootb`, so I'm fine moving forward with that.

**Comment by menski** [comment]
Changes are in place:
- https://github.com/camunda/connectors-bundle
- https://hub.docker.com/repository/docker/camunda/connectors-bundle
